### ISSUE TITLE
Cactus: use a collection for input

### ISSUE THREAD

**Issue by TomHarrop** [galaxyproject/tools-iuc#5752]
As suggested by @bernt-matthias in https://github.com/galaxyproject/tools-iuc/pull/5741. Using a collection as input would allow this tool to be used with workflows. I'll work on it.

**Comment by bgruening** [comment]
@TomHarrop are you planning to work on it?

**Comment by TomHarrop** [comment]
Yes, but I haven't started yet, so if someone else is keen to do it that's fine.
